src/pages/studyView/tabs/SummaryTab.tsx

```diff
@@ -254,29 +238,35 @@ export class StudySummaryTab extends React.Component<
                     this.store.resetGeneFilter(chartMeta.uniqueKey);
                 props.selectedGenes = this.store.selectedGenes;
                 props.onGeneSelect = this.store.onCheckGene;
-                (props.title =
-                    props.title +
-                    (!this.store.molecularProfileSampleCounts.isComplete ||
-                    this.store.molecularProfileSampleCounts.result === undefined
-                        ? ''
-                        : ` (${this.store.molecularProfileSampleCounts.result.numberOfCNAProfiledSamples} profiled samples)`)),
-                    (props.getData = () =>
-                        this.store.getGenesCNADownloadData());
+                props.title = this.store.getChartTitle(
+                    ChartTypeEnum.CNA_GENES_TABLE,
+                    props.title
+                );
+                props.getData = () => this.store.getGenesCNADownloadData();
                 props.genePanelCache = this.store.genePanelCache;
                 props.downloadTypes = ['Data'];
                 props.filterByCancerGenes = this.store.filterCNAGenesTableByCancerGenes;
                 props.onChangeCancerGeneFilter = this.store.updateCNAGenesTableByCancerGenesFilter;
                 break;
             }
-            case SURVIVAL: {
+            case ChartTypeEnum.GENOMIC_PROFILES_TABLE: {
```

Comment:
No title for this table?

Reply:
There is a default title for it `'Genomic Profile Sample Counts'` at https://github.com/cBioPortal/cbioportal-frontend/pull/3090/files#diff-48cb532a65b28af956576b588b2ce39fR181